tests/data/test_preprocess_cord.py

```diff
@@ -6,18 +6,22 @@
 import unittest
 from datetime import datetime
 
-# from contradictory_claims.data.preprocess_cord import filter_metadata_for_covid19
+import pandas as pd
+from contradictory_claims.data.preprocess_cord import clean_text, construct_regex_match_pattern,\
+    extract_json_to_dataframe, extract_regex_pattern, filter_metadata_for_covid19
 
-from .constants import metadata_path, pub_date_cutoff
+from .constants import pdf_filenames, pmc_filenames, pub_date_cutoff,\
+    sample_covid19_df_path, sample_json_temp_path, sample_json_text_file_dir,\
+    sample_metadata_path, sample_virus_lex_path
 
 
 class TestPreprocessCord(unittest.TestCase):
     """Tests for preprocessing CORD-19."""
 
     def test_find_files(self):
         """Test that input files are found properly."""
-        self.assertTrue(os.path.isfile(metadata_path),
-                        "Metadata.csv not found at {}".format(metadata_path))
+        self.assertTrue(os.path.isfile(sample_metadata_path),
+                        "Metadata.csv not found at {}".format(sample_metadata_path))
 
     def test_pub_date_cutoff(self):
         """Test that publish date cut-off is in the correct format."""
```

Comment:
Hmmm there should be a way at runtime that we're testing the user enters in a correct date format. Maybe entering date is an optional flag, at which point a test like this is run, otherwise there's a default date cutoff (i.e. no cutoff probably)

Reply:
Yeah good point, done